src/controller.coffee

```diff
@@ -140,6 +140,8 @@ class Controller
         @trigger "df:results:error", [err]
       else if res.query_counter == @queryCounter
         @lastPage = Math.ceil (res.total / res.results_per_page)
+        # save the las custom_results_id received
+        @custom_results_id = res.custom_results_id
```

Comment:
@sariogonfer this shouldn't go here... this is a job for the layers. You can paint the custom results id in the results template.